tests/e2e/squid_ssl_bump_functionality_test.go

```diff
@@ -27,7 +27,7 @@ var _ = Describe("Squid SSL-Bump Functionality", Ordered, func() {
 		squidPod      *corev1.Pod
 	)
 
-	const testServerURL = "https://test-server.proxy.svc.cluster.local:443"
+	const testServerURL = "https://test-server.caching.svc.cluster.local:443"
```

Comment:
Why not use the `namespace` variable to help define this url?

There are a couple other places below where the same hostname is hard coded.